aws_list_all/apply_filter.py

```diff
@@ -0,0 +1,167 @@
+import json
+import os
+import sys
+
+from .fixing_filter import *
+from .resource_filter import *
+
+def apply_filters(listing, unfilter, response, complete):
+    """Apply filters for operations to be handled in a special way or 
+        to remove default resources from the response"""
+    unfilterList = convert_unfilterList(unfilter)
```

Comment:
As far as I can see, with action="append", unfilter will always be a list already?